bower.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
can you also update the entry point main property?
```json
  "main": [
     "./dist/js/materialize.clockpicker.js",
     "./dist/css/materialize.clockpicker.css"
   ],
```

this is also useful when injecting files to index.html e.g when using ```gulp-inject```